weave-and-docker-platform/2-machine-weaveproxy-swarm.md

```diff
@@ -92,54 +92,53 @@ and avoid having to set the <code>--init-peer-count</code> explicitly. In this e
 
 ##Launching Weave
 
-Next launch Weave onto the virtual machines.
+If you are continuting on from Part 1 of this guide, then first stop weave on weave-1: 
 
-The IP addresses of all the peers are not known ahead of time, so you will need to pass `--init-peer-count 3` to `weave launch`.
-`--init-peer-count` is set to 3, as we are specifying a cluster of 3 VMs.
-
-On each host you will need to:
+~~~bash
+weave stop
+weave env --restore
```

Comment:
You would need to do `eval $(weave env --restore)`, it doesn't do anything otherwise.